backport_packages/setup_backport_packages.py

```diff
@@ -35,9 +36,20 @@
 
 from setuptools import Command, find_packages, setup as setuptools_setup
 
-from backport_packages.import_all_provider_classes import import_all_provider_classes
-from setup import PROVIDERS_REQUIREMENTS
-from tests.deprecated_classes import HOOKS, OPERATORS, SECRETS, SENSORS, TRANSFERS
+MY_DIR_PATH = os.path.dirname(__file__)
+SOURCE_DIR_PATH = os.path.abspath(os.path.join(MY_DIR_PATH, os.pardir))
+AIRFLOW_PATH = os.path.join(SOURCE_DIR_PATH, "airflow")
+PROVIDERS_PATH = os.path.join(AIRFLOW_PATH, "providers")
+
+sys.path.insert(0, SOURCE_DIR_PATH)
+
+# isort:skip
+from setup import PROVIDERS_REQUIREMENTS  # noqa
+
+# isort:skip
+import tests.deprecated_classes  # noqa
+# isort:skip
+from backport_packages.import_all_provider_classes import import_all_provider_classes  # noqa
```

Comment:
Would you mind adding a comment why such order? One day someone may wonder about that

Reply:
Good point. Added.